newdle/client/src/components/UserSearch.js

```diff
@@ -46,9 +50,9 @@ export default function UserSearch() {
   );
 
   const handleModalConfirm = useCallback(() => {
-    setParticipants([...participants, ...stagedParticipants]);
+    dispatch(updateParticipantList([...participants, ...stagedParticipants]));
```

Comment:
I think this should just pass the staged ones and leave it up to the reducer to merge the old/new list

Reply:
But then I would also have to handle it differently when removing a participant from the list. If I do it like that then I can handle all cases with one action.

Reply:
I think having separate actions to add/remove participants is cleaner...